lib/main.js

```diff
@@ -313,16 +321,22 @@ const Hydrogen = {
   },
 
   createResultBubble(editor: atom$TextEditor, code: string, row: number) {
-    if (!store.grammar) return;
+    const { grammar, filePath, kernel } = store;
+    if (!grammar || !filePath) return;
```

Comment:
> Did you consider using the editor id? This would handle filename changes as well.

As long as we also show the name of the file when it is named, so user's know which are which. I hate looking at a pile of UUIDs.

Reply:
Interesting point, I never use Hydrogen with unsaved files since it requires me to manually set a grammar first.

Unfortunately the editor id changes when a editor tab is closed and reopened so it's not really suitable for out use case. 

> As long as we also show the name of the file when it is named, so user's know which are which. I hate looking at a pile of UUIDs.

👍 I agree.

Reply:
> As long as we also show the name of the file when it is named, so user's know which are which. I hate looking at a pile of UUIDs.

@rgbkrk Does this mean I can't count on you subscribing to my new podcast where I talk about all the hot new trending UUID's?